README.md

```diff
@@ -1,5 +1,5 @@
 # Plex Auto Collections
-##### Version 2.1.1
+##### Version 2.2.0
```

Comment:
in plex_auto_collections.py right after the big title it prints out there's another version number

Reply:
Will also update this.